src/Features/LanguageServer/ProtocolUnitTests/CodeActions/CodeActionsTests.cs

```diff
@@ -41,6 +41,9 @@ void M()
                 kind: CodeActionKind.Refactor,
                 children: Array.Empty<LSP.VSCodeAction>(),
                 data: CreateCodeActionResolveData(CSharpAnalyzersResources.Use_implicit_type, caretLocation),
+                priority: PriorityLevel.Normal,
+                groupName: "Roslyn1",
```

Comment:
Specifying "Roslyn1" seems to work for now, but may be a little risky in the future if the first code action in the list changes to something else in the future.